From b70023409a8ab0ab43af6f7651bf27e7f531b3ba Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 14 Oct 2018 18:25:26 +0300
Subject: [PATCH] Makefile: release: make the jenkins job name variable

Move the jenkins job name as variable instead of hard coding.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 30cad10..15c1474 100644
--- a/Makefile
+++ b/Makefile
@@ -75,6 +75,7 @@
 	sed 's/origin\///' | sort -n | tail -n 1)
 
 RELEASE_VER_COMPAT := $(patsubst v%,%,$(RELEASE_BRANCH))
+RELEASE_JOB_NAME := goya-release-test
 RELEASE_JOB_BRANCH := "package_branch=$(RELEASE_BRANCH)"
 RELEASE_JOB_VER_COMPAT := "release_version=$(RELEASE_VER_COMPAT)"
 RELEASE_JOB_URL := http://192.168.0.4:8080/job/goya-release-test/buildWithParameters
@@ -122,7 +123,7 @@
 	@repo forall $(RELEASE_REPOS) -c git push origin $(EMBD_FULL_VER) $(RELEASE_TAG_POINTER)
 	@echo "Released: $(RELEASE_BRANCH) $(EMBD_FULL_VER)"
 	@wget -O - "$(RELEASE_JOB_URL)?$(RELEASE_JOB_PARAMS)"
-	@echo "Started the \"goya-release-test\" jenkins job... check your email for results."
+	@echo "Started the \"$(RELEASE_JOB_NAME)\" jenkins job... check your email for results."
 
 .PHONY: hl-goya hl-gaudi
 hl-goya hl-gaudi: $(OUTPUT_DIR)
